product-backend/src/test/java/test/JobRoleServiceTest.java

```diff
@@ -0,0 +1,73 @@
+package test;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.api.JobRoleService;
+import org.kainos.ea.client.FailedToGetAllJobRolesException;
+import org.kainos.ea.db.JobRoleDao;
+import org.kainos.ea.cli.JobRole;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+@ExtendWith(MockitoExtension.class)
+public class JobRoleServiceTest {
```

Comment:
- lack of formatting
- you are actually mocking the real service you want to test, you should mock service dependencies